_includes/tutorials/column-difference/ksql/markup/dev/transient-reporting.adoc

```diff
@@ -0,0 +1,31 @@
+Now we write a query to concatenate multiple columns. To achieve this, we will use the `+` operator between the fields in our `SELECT` statement rather than a comma.
```

Comment:
I think this is from the tutorial you copied, and that you should be talking about difference and `-` here instead of concatenate & `+`.

Reply:
Maybe right here would be a good place to mention the numeric comment you had in the "Short Answer" section. I could see people skipping over the "Short Answer" and missing that key piece of info. Just a thought.